plugins/source/aws/codegen/recipes/config.go

```diff
@@ -60,6 +60,25 @@ func ConfigResources() []*Resource {
 					},
 				}...),
 		},
+		{
+			SubService: "config_rules",
+			Struct:     new(types.ConfigRule),
+			ExtraColumns: append(defaultRegionalColumns,
+				[]codegen.ColumnDefinition{
+					{
+						Name:     "arn",
+						Type:     schema.TypeString,
+						Resolver: `schema.PathResolver("ConfigRuleArn")`,
+						Options:  schema.ColumnCreationOptions{PrimaryKey: true},
+					},
+				}...),
+			Relations: []string{"ConfigRuleCompliances()"},
+		},
+		{
+			SubService:   "config_rule_compliances",
+			Struct:       new(types.ComplianceByConfigRule),
+			ExtraColumns: defaultRegionalColumns,
```

Comment:
I guess not, but are there any good primary key candidates we can use for this table?

Reply:
I thought we didn't put PKs on relation tables

Reply:
We do if there are good candidates, because it's more reliable than `_cq_id`. But if there are no good candidates, we don't.